xbmc/cores/DataCacheCore.h

```diff
@@ -298,6 +318,11 @@ class CDataCacheCore
     float m_tempo;
     float m_speed;
     bool m_frameAdvance;
+    /*! Time point of the last seek operation */
+    std::chrono::time_point<std::chrono::system_clock> m_lastSeekTime{
+        std::chrono::time_point<std::chrono::system_clock>{}};
```

Comment:
is it enough to initialize with just `std::chrono::time_point<std::chrono::system_clock> m_lastSeekTime{}; ?`
no idea about it, just asking ;)

Reply:
as far as I understood that sets the time_point to epoch